associations.rb

```diff
@@ -46,44 +53,59 @@ def generate_migrations
 
 
 class Hotel < ActiveRecord::Base
-  #insert our associations here
- 
+  has_many :rooms
+
   def to_s
     "#{name} with #{rooms.count} rooms"
   end
-end
 
-class Booking < ActiveRecord::Base
-  #insert our associations here
+  def bookings
+    Booking.where(:room => rooms.map{|r| r.id})
```

Comment:
I'm pretty sure you don't need to map the rooms to their ids. ActiveRecord probably handles this the same:

```ruby
Booking.where(:room => rooms)
```

Better yet, let ActiveRecord handle the whole thing:

```ruby
has_many :bookings, through: :rooms
```